docs-next/pages/tutorials/getting-started-with-create-keystone-next-app/index.mdx

```diff
@@ -0,0 +1,134 @@
+import { Markdown } from '../../../components/Page';
+import Image from 'next/image';
+import Link from 'next/link';
+
+<img
+  src={require('./assets/cover.svg')}
+  alt="A terminal with the output of create-keystone-next-app"
+  style={{
+    margin: '0 -1rem 2rem -1rem',
+    width: 'calc(100% + 2rem)',
+    maxWidth: 'none',
+  }}
+/>
+
+# Getting started with `create-keystone-next-app`
+
+[`create-keystone-next-app`](https://github.com/keystonejs/create-keystone-next-app) is a CLI app that makes it easier for you to get started with Keystone.
+It will generate a couple files for you and install all the dependencies you need to run the AdminUI and start using the GraphQL API.
+
+## Quick Start
+
+```sh
+yarn create keystone-next-app
+cd my-app
+yarn dev
+```
+
+## Installing your own Keystone instance
+
+Open your preferred shell and make sure you are in the folder you would like to create your new project in.
+`create-keystone-next-app` will generate a new folder with your new Keystone files in it.
+
+### Yarn
+
+Use [`yarn create`](https://classic.yarnpkg.com/en/docs/cli/create/):
+
+```sh
+cd your/path/
+yarn create keystone-next-app
+```
+
+### npm
+
+Use [`npm init`](https://docs.npmjs.com/cli/v7/commands/npm-init):
+
+```sh
+npm init keystone-next-app
+```
+
+_(`npm init <initializer>` is available in npm 6+)_
+
+### npx
+
+Use npms [`npx`](https://docs.npmjs.com/cli/v7/commands/npx):
+
+```sh
+cd your/path/
+npx create-keystone-next-app
+```
+
+_(npx comes with npm 5.2+)_
+
+The CLI will ask you to name your new app and for a way to connect to your database.
+Make sure you have your connection url ready for your database like e.g. `postgres://localhost/my-app` or `postgres://username:password@localhost/my-app`.
+
+_(If you run into trouble with the database, check out this guide on [Database Setup](https://www.keystonejs.com/quick-start/adapters))_
+
+## Opening your shiny new AdminUI
```

Comment:
```suggestion
## Opening your shiny new Admin UI
```
I'm not a big fan of this style of language, it makes it all a bit... I dunno. I would go with `Opening the Admin UI` or something less cutesy. Not gonna block on that though.